### ISSUE TITLE
[🕹️] Completed All side quests of 20crm

### ISSUE THREAD

**Issue by adilkadivala** [twentyhq/twenty#7933]
### What side quest or challenge are you solving?

All side quests I have Completed 

### Points

**Like and retweet** : 50 points
**Share a tweet about your favorite feature in Twenty** : 50 points
**Create a bug report** : 50-150 points
**Meme Magic** : 50 Points
**Gif Magic** : 50 Points

**🧙 Quest Wizard** :  300 extra points

### Description

I have completed all side quest,  create bug report's issue I have open in bug template 

here it's link

https://github.com/twentyhq/twenty/issues/7932

### Provide proof that you've completed the task

Like and retweet

![newfirst](https://github.com/user-attachments/assets/c1c4a062-f727-4a03-84ee-66c23ba56907)

tweet about your favorite feature in Twenty

![xtwentypost](https://github.com/user-attachments/assets/37dfeaf3-9808-4909-b065-d927edeb8d2b)

Create a bug report

[issue open here](https://github.com/twentyhq/twenty/issues/7932)

Meme Magic

![twenty meme](https://github.com/user-attachments/assets/0d892dac-0d28-4366-9cd8-dcc2dd180af7)

Gif Magic

[Gif](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExNXNkcXgyYzMxMnA1bmU2eDBrbXg5cWdhamYwdTM3ZWl3aGkxZjl5YSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/GjSIGfA9DjeJnKaIj7/giphy-downsized-large.gif)

[gif2](https://giphy.com/gifs/GjSIGfA9DjeJnKaIj7)

**Comment by adilkadivala** [comment]
Hi @Bonapara 

I have completed all side quests,

Here is mine x , LinkedIn profile 

I have followed twenty on both side

**X**
https://x.com/adil_kadival

**LinkedIn**

https://www.linkedin.com/in/adilkadival

**retweet post **
https://x.com/ossgoodgame/status/1841019549183066552

**created post about twenty on X**
https://x.com/adil_kadival/status/1848550486137573805

**I have find a bug** 
here it is
https://github.com/twentyhq/twenty/issues/7932#issuecomment-2429510802

and also created meme gif, 

so in short I have completed this all task 
![maintwentyside](https://github.com/user-attachments/assets/1de7d6d0-68cb-4f1c-82d0-5b1f667823cb)


so total points should be **550** with **250 for tasks** and **300 for completing all task**. so can you please recheck it?
